src/components/ebay-fake-menu/examples/06-a11y-current-value/template.marko

```diff
@@ -0,0 +1,5 @@
+<ebay-fake-menu>
+    <@item href="#" current a11y-current-value='true'>item 1</@item>
```

Comment:
I thought we were going with enhancing `current` to be an enumerated type, and falling back to a boolean for backwards compatibility? If we are sticking with two attributes, my preference would be `itemMatchesUrl` to be consistent with what we did in tabs.

Reply:
I think `current` should stay as a boolean. We have booleans in other places where it makes sense and there can only be 1 possible value. HTML has the same. 
I do think that either:
`a11y-current-value` should be on the parent though if we are going this route.
If we do `current='true|page'` that can work too, but it might be confusing what the default would be as well as overloading both a boolean/string. 

My personal preference is keep current as a boolean, set the default on the `<ebay-fake-menu>`. I don't care about the variable name as much, but I think an enum makes sense because we can potentially have other values in the future.

Reply:
A problem with `a11y-current-value` is that it implies that any [aria-current](https://www.w3.org/TR/wai-aria-1.1/#aria-current) value can be used. Really, it's only every going to be "true" or "page" in this case. Probably not a huge deal, and we could clear it up in our docs.

A potential problem of putting this on the root element is that for some fake menus there might be a mix of items where some should have `page` and some should have `true`. Definitely getting into edge cases here, but it is a possible scenario.

Reply:
Hmm I guess I don't quite understand what you're saying about the root element. The `a11y-current-value` is applied on each item, not on the `fake-menu`. So that should allow the user to configure each item individually

Reply:
Andrew was proposing moving it from each item to the item parent, but I foresee a problem with that.

I think for the sake of moving things forward, I recommend just going with:

`<@item href="#" current item-matches-url='false'>item 1</@item>`

This unblocks us and aligns with what we currently have in fake-tabs. We can come back to this and make some breaking changes and cleanup in v8.